README.md

```diff
@@ -7,12 +7,20 @@
 
 ## What is it?
 
-iTowns is a web framework written in Javascript/WebGL for visualisation of 3D geographic data allowing precise measurements in 3D.
-Its first purpose was the visualisation of street view images and terrestrial lidar point cloud. It has then been extended to handle more data types.
+iTowns is a [Three.js](https://threejs.org/)-based framework written in Javascript/WebGL for visualizing 3D geospatial data allowing precise measurements.
```

Comment:
The "allowing precise measurements" part seems a bit odd to men, since we don't have measuring tools yet.